src/index.js

```diff
@@ -0,0 +1,181 @@
+import './style.css';
+import { countElements } from './comments.js';
+
+const getLink = 'https://api.nasa.gov/planetary/apod?api_key=4SuCuNxM0J6w1FmjQokTcawsubomH7aV4ep60VgT&start_date=2021-03-16&end_date=2021-03-21';
+const datesForPopup = ['2021-03-16', '2021-03-17', '2021-03-18', '2021-03-19', '2021-03-20', '2021-03-21'];
+const getImage = (num) => `https://api.nasa.gov/planetary/apod?api_key=4SuCuNxM0J6w1FmjQokTcawsubomH7aV4ep60VgT&date=${datesForPopup[num]}`;
+const main = document.getElementById('addToScreen');
+const starLink = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/9mAPgvMc6PjOJk4JU1ZU/likes';
+const commentLink = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/9mAPgvMc6PjOJk4JU1ZU/comments';
+
+const getScores = async (url) => {
+  const response = await fetch(url);
+  return response.json();
+};
+
+const postScores = async (url, data) => {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+  return response;
+};
+
+function addToScoreBord(img, title, index) {
+  const div = document.createElement('div');
+  div.classList.add('cardContainer');
+  div.innerHTML = `
+        <div class="imgcontainer">
+          <img src="${img}" alt="">
+        </div>
+        <div class="title">
+          <h3>${title}</h3>
+          <a href="#" id="${index}star" class="stars"><i class="far fa-star"></i></a>
+        </div>
+        <small></small>
+        <input type="button" value="Comments" id="${index}" class="comment">
+        `;
+  main.appendChild(div);
+}
+
+function closePopup(target) {
+  target.parentElement.parentElement.parentElement.remove();
+}
+
+function displayPopup(img, title, description, id) {
+  const popupDiv = document.createElement('div');
+  popupDiv.classList.add('popupWindow');
+  popupDiv.innerHTML = `
+  <div class="popupContainer">
+   <span><i class="fas fa-times" id="close"></i>
+   </span>
+   <div class="header-popup">
```

Comment:
- [ ] Kindly change these and all other regular functions to es6 arrow function syntax as stated in the Javascript best practices guide.

Reply:
- [ ] The issue has not been properly resolved

Reply:
According to this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) js best practices do not ask to use arrow functions, using es6 syntax was put in learning objectives only